ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PauseResumeIntegrationTest.java

```diff
@@ -188,36 +192,37 @@ public void pausedQueriesShouldBePausedOnRestart()  {
     createQuery("1");
     String queryId = ((Queries) RestIntegrationTestUtil.makeKsqlRequest(REST_APP, "SHOW "
             + "QUERIES;").get(0)).getQueries().get(0).getId().toString();
-    createQuery("2");
+//    createQuery("2");
 
     TEST_HARNESS.produceRows(PAGE_VIEW_TOPIC, PAGE_VIEWS_PROVIDER, KAFKA, JSON, System::currentTimeMillis);
 
     Supplier<Integer> supplier = getSupplier(1);
-    Supplier<Integer> supplier2 = getSupplier(2);
+//    Supplier<Integer> supplier2 = getSupplier(2);
 
     // 7 records are produced + a header & footer.
     assertThatEventually(supplier, equalTo(9));
-    assertThatEventually(supplier2, equalTo(9));
+//    assertThatEventually(supplier2, equalTo(9));
 
     // When:
-    RestIntegrationTestUtil.makeKsqlRequest(REST_APP, "PAUSE " + queryId + ";");
-    assertThatEventually(this::getPausedCount, equalTo(1L));
+    //RestIntegrationTestUtil.makeKsqlRequest(REST_APP, "PAUSE " + queryId + ";");
+    //assertThatEventually(this::getPausedCount, equalTo(1L));
     assertThatEventually(supplier, equalTo(9));
 
     // Restart server
     REST_APP.stop();
```

Comment:
@wcarlson5 After the updates to this unit test, the server is just restarted and there's no pausing/resuming happening.  After the restart, the query is restarted from the beginning causing duplicate data in the output topic.